doc/v2/documentation_data_format.rst

```diff
@@ -0,0 +1,1172 @@
+  The ``order`` value should be used if values of the independent variable are
+  ordinal. This value can only be used in combination with ``LinePlot`` value for
+  the ``plotTypeSimulation`` column. In this case, points on x axis will be
+  placed equidistantly from each other. Default is ``lin``.
+
+- ``yValues`` [observableId, REFERENCES(measurementTable.observableId), OPTIONAL]
+
+  The observable which should be plotted on the y-axis.
+
+- ``yOffset`` [NUMERIC, OPTIONAL]
+
+  Possible data-offsets for the observable (default is ``0``).
+
+- ``yLabel`` [STRING, OPTIONAL]
+
+  Label for the y-axis. Defaults to the entry in ``yValues``.
+
+- ``yScale`` [STRING, OPTIONAL]
+
+  Scale of the observable, can be ``lin``, ``log``, or ``log10``. Default is ``lin``.
+
+- ``legendEntry`` [STRING, OPTIONAL]
+
+  The name that should be displayed for the corresponding dataset in the
+  legend and which defaults to the value in ``datasetId``.
+
+
+Mapping table
+-------------
+
+Mapping PEtab entity IDs to entity IDs in the model. This optional file may be
+used to reference model entities in PEtab files where the ID in the model would
+not be a valid identifier in PEtab (e.g., due to inclusion of blanks, dots, or
+other special characters).
+
+The TSV file has two mandatory columns, ``petabEntityId`` and
+``modelEntityId``. Additional columns are allowed.
+
++---------------+---------------+
+| petabEntityId | modelEntityId |
++===============+===============+
+| STRING        | STRING        |
++---------------+---------------+
+| reaction1_k1  | reaction1.k1  |
++---------------+---------------+
+
+
+Detailed field description
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- ``petabEntityId`` [STRING, NOT NULL]
+
+  A valid PEtab identifier that is not defined in any other part of the PEtab
+  problem. This identifier may be referenced in condition, measurement,
+  parameter and observable tables, but cannot be referenced in the model
+  itself.
+
+- ``modelEntityId`` [STRING, NOT NULL]
+
+  A globally unique identifier defined in the model,
+  *that is not a valid PEtab ID* (see :ref:`identifiers`).
+
+  For example, in SBML, local parameters may be referenced as
+  ``$reactionId.$localParameterId``, which are not valid PEtab IDs as they
+  contain a ``.`` character. Similarly, this table may be used to reference
+  specific species in a BNGL model that may contain many unsupported
+  characters such as ``,``, ``(`` or ``.``. However, please note that IDs must
+  exactly match the species names in the BNGL-generated network file, and no
+  pattern matching will be performed.
+
+Extensions
+~~~~~~~~~~
+
+Additional columns, such as ``Color``, etc. may be specified. Extensions
+that define operations on multiple PEtab problems need to employ a single
+PEtab YAML file as entrypoint to the analysis. This PEtab file may leave all
+fields specifying files empty and reference the other PEtab problems in the
+extension specific fields.
```

Comment:
Is this useful? PEtab Select currently doesn't do this, and I don't see any benefits from doing this. It would just add an additional file to PEtab Select that duplicates information in the Model Space table.

Reply:
Or by "operations on multiple PEtab problems", do you mean rather, estimating parameters across multiple PEtab problems? Is there a use case? If not, it could be removed from the spec...

Reply:
It's rather about the petab-select case. But I don't think it makes sense to consider petab-select an extension in the sense used here. It just builds on top of PEtab problems, but doesn't really change the interpretation of any specific PEtab problem. I think it could be removed, but maybe that's a separate discussion.
I think the main point here was that for a PEtab problem, we always want to have a yaml file that lists the extensions.

Reply:
I agree re: putting extensions that modify a PEtab problem, into the PEtab YAML. Since I don't yet see a use case for "Extensions that define operations on multiple PEtab problems", I don't understand/can't review this part of the spec.